doc/nrf/libraries/networking/lwm2m_location_assistance.rst

```diff
@@ -41,6 +41,74 @@ There are three different supported methods of obtaining the location assistance
 * Query of P-GPS predictions - The P-GPS predictions are queried from nRF Cloud and provided back to the device through the LwM2M server.
   The predictions are stored in the device flash and injected to the GNSS module when needed.
 
+API usage
+*********
+
+Cell based location
+===================
+
+Cell based location can be based only on current cell or current and neighboring cells.
+Getting the information about the current cell is achieved by using the Connectivity Monitor object (ID 4).
+By enabling the kconfig option :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_CONN_MON_OBJ_SUPPORT`, the connectivity monitor is populated with data about current cell.
+
+Getting the information about the neighboring cells is achieved by using a collection of ECID-Signal Measurement Information object (ID 10256).
+These are populated by calling :c:func:`lwm2m_ncell_handler_register` to register the listener for the neighborhood measurements and scheduling a measurement by calling :c:func:`lwm2m_ncell_schedule_measurement`.
```

Comment:
```suggestion
To get information about the neighboring cells, use a collection of ECID Signal Measurement Information objects (ID 10256).
To populate the objects, call the :c:func:`lwm2m_ncell_handler_register` function to register the listener for the neighborhood measurements and :c:func:`lwm2m_ncell_schedule_measurement` to schedule a measurement.
```

Reply:
ECID-Signal Measurement Information is intended as the object is named that in the registry